posix.mak

```diff
@@ -519,7 +523,17 @@ test:
 # Changelog generation
 ################################################################################
 
-pending_changelog:
-	@echo "This command will be available soon."
+changelog/${NEXT_VERSION}.dd: ${STABLE_DMD} ../tools ../installer
+	$(STABLE_RDMD) $(TOOLS_DIR)/changed.d "v${LATEST}..upstream/stable" -o changelog/${NEXT_VERSION}.dd \
```

Comment:
This was already changed, as we realized that showing the bug list from Bugzilla will lead to unrelated diffs on DAutoTest: https://github.com/dlang/dlang.org/pull/1588